public/index.js

```diff
@@ -66,15 +129,19 @@ darkLightButton.addEventListener("change", () => {
     allergies,
     headline,
     userWantAnotherRecipe,
+    tryAgainBtn,
     ...recipeButtons,
   ].forEach((element) => {
     element.style.setProperty("--green", color);
     element.style.transition = "background-color 0.5s ease";
   });
 });
+
 recipeButtons.forEach((button) => {
   console.log(userText.value)
-  button.addEventListener("click", () => {
+  button.addEventListener("click", async () => {
```

Comment:
This is partly to do with the other feature I have been working on regarding both recipe and image loading at the same time. It's not done though yet.